plugins/swf/src/hooks/useServerlessWorkflowCombinedEditor.ts

```diff
@@ -34,9 +34,15 @@ import { empty_definition } from '@backstage/plugin-swf-common';
 import { SwfLanguageService } from './SwfLanguageService';
 import { Diagnostic, DiagnosticSeverity } from 'vscode-languageserver-types';
 import { Notification } from '@kie-tools-core/notifications/dist/api';
-import { SwfTextEditorChannelApiImpl } from './SwfTextEditorChannelApiImpl';
-import { ServerlessWorkflowTextEditorChannelApi } from '@kie-tools/serverless-workflow-text-editor/dist/api';
 import { SwfLanguageServiceChannelApiImpl } from './SwfLanguageServiceChannelApiImpl';
+import { parseApiContent } from '@kie-tools/serverless-workflow-service-catalog/dist/channel';
+import { SwfCatalogSourceType } from '@kie-tools/serverless-workflow-service-catalog/dist/api/types';
+import { SwfServiceCatalogService } from '@kie-tools/serverless-workflow-service-catalog/dist/api';
+import { SwfCombinedEditorChannelApiImpl } from '@kie-tools/serverless-workflow-combined-editor/dist/impl';
+import { ServerlessWorkflowCombinedEditorChannelApi } from '@kie-tools/serverless-workflow-combined-editor/dist/api';
+
+// This filename should probably be returned by the backend
+const ACTIONS_OPEN_API_FILE = 'actions-openapi.json';
```

Comment:
We use this file name [here](https://github.com/manstis/backstage/blob/7d98b25009d1246d01526bbcff64bd4c18e074d3/plugins/swf-backend/src/service/WorkflowService.ts#L55) too.

Could you please move this constant to [here](https://github.com/manstis/backstage/blob/7d98b25009d1246d01526bbcff64bd4c18e074d3/plugins/swf-common/src/constants.ts) and use it in both the place referenced above and your change here?